java/memory/memory-netty-buffer-patch/src/test/java/io/netty/buffer/TestUnsafeDirectLittleEndian.java

```diff
@@ -70,4 +57,22 @@ public void testPrimitiveGetSet() {
       e.printStackTrace();
     }
   }
+
+  private static UnsafeDirectLittleEndian getUnsafeDirectLittleEndian(ByteBuf byteBuf) {
```

Comment:
Just curious, was there a reason to split this out?

Reply:
Readability / Modularity. Now that I look at it a day later, it's probably more of the former than the latter! Sorry about that. I probably should have asked before doing that.

Reply:
I guess that's okay. But we don't use it any other place?

Reply:
It's not a big deal, just wanted to understand if there was another reason.